pkgs/default.nix

```diff
@@ -14,6 +14,13 @@
     .${wine};
 
   packages = rec {
+    dxvk = callPackage ./dxvk {
```

Comment:
Overriding just the target inputs from pkgsCross is hacky but I don't know the right way to cross compile a single package outside of nixpkgs.

If this was an overlay you'd just do `callPackage ./dxvk { }` and then would build `pkgs.pkgsCross.mingwW64.dxvk` or `pkgs.pkgsCross.mingw32.dxvk` instead of defining it twice like this.

Reply:
So it looks like unless you're using an overlay cross compiling isn't properly supported in flakes so it has to be done this hacky way. See https://discourse.nixos.org/t/how-do-i-cross-compile-a-flake/12062/18?u=lun

Reply:
I could also change the flake's .packages output to use the overlay properly as an overlay and actually use the final packages and reduce the hackiness in this file.

Either way is hacky it just shifts the hacky around :/

Reply:
Managed to get it working with less hacks, just using callPackage from pkgsCross.